changelog.mdx

```diff
@@ -5,6 +5,29 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## January 6, 2025
+
+## Reduced cloud costs from optimized resource usage with KEDA
+
+We’ve implemented [**KEDA (Kubernetes-based Event Driven Autoscaler)**](https://keda.sh/) and are rolling it out to all customer clusters over the next few weeks.
+With KEDA, your environment now dynamically scales up and down--even to **zero machines** when there’s reduced or no workload.
+With automatic scaling we're able to optimize resource usage and reduce cloud costs--expect to see a drop in your cloud spend as a result.
+
+## View logs and export metrics using the CLI
+
+View logs using the Chalk CLI; inspect your log stream the same way you would from the Chalk dashboard but via the command line.
+`chalk logs [--query="the same string you can put in the web ui"]`
+In addition to exporting metrics with a custom OpenMetrics compatible collector, you can now also export metrics directly using the CLI with `chalk metrics export`.
```

Comment:
@emarx Can we actually export the metrics using the CLI or is it primarily viewing them?
I'll update this phrasing if the latter